package.json

```diff
@@ -1,9 +1,13 @@
 {
-  "name": "apollo-cursor-pagination",
-  "version": "0.9.1",
-  "description": "Relay's Connection implementation for Apollo Server GraphQL library",
+  "name": "@pocket-tools/apollo-cursor-pagination",
+  "version": "1.0.0",
+  "description": "Relay's Connection implementation for Apollo Server GraphQL library with primary key support for cursor generation.",
+  "keywords": [
+    "apollo",
+    "pagination"
+  ],
   "main": "dist/index.js",
-  "repository": "https://github.com/Terminal-Systems/apollo-cursor-pagination",
+  "types": "dist/index.d.ts",
```

Comment:
We have no types

Reply:
could be old diff.. it seems like removed and i dont see it in the package.json,

Reply:
Maybe you didn't push local changes? I'm still seeing it on this PR after refresh.

Reply:
lol now it shows as updated even if the only change was with README file

Reply:
are you seeing it now?

Reply:
no i am not..it shows as `outdated`

Reply:
oh i see, the `types` got removed in this commit: https://github.com/Pocket/apollo-cursor-pagination/pull/5/commits/5b553f9e4d93ed8700f5648aa32c3db25fafd1a2 but somehow its not showing up in your UI, confused why

Reply:
it's all good now!